cryptofeed/backends/quasar.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timedelta
+import quasardb
+import quasardb.pandas as qdbpd
+from cryptofeed.backends.backend import (BackendBookCallback,BackendCallback,BackendQueue)
+
+class QuasarCallback(BackendQueue):
+    def __init__(self, host="127.0.0.1", port=2836, username:str="", private_key:str="", public_key:str="", none_to=None, shard_size:timedelta = timedelta(minutes=15)):
+        self.numeric_type = float
+        self.table = ""
+        self.running = True
+        self.url = f"qdb://{host}:{port}"
+        self.none_to = none_to
+        self.columns=[]
+        self.shard_size = shard_size
+        
+        with quasardb.Cluster(self.url, user_name=username, user_private_key=private_key, cluster_public_key=public_key) as conn:
+            self.conn = conn
```

Comment:
I think this does not work; if it does, it should not. Because the scope of the connection is short-lived, so it would close the connection as soon as this line of code is executed. 

So I don't understand how this could work?

Reply:
Indeed it shouldnt work, i will move it to write method

Reply:
How often is `write` invoked? Because establishing a connection is non-trivial. There's also a global connection pool API for python, perhaps it makes more sense to use in the `write` method?

Reply:
write is invoked on every channel update e.g. each new trade invokes write
connection pool is now created at initialization and `write` connects to the pool